plugin/api.go

```diff
@@ -52,6 +52,9 @@ type API interface {
 	// GetUserByUsername gets a user by their username.
 	GetUserByUsername(name string) (*model.User, *model.AppError)
 
+	// GetUsersByUsernames gets a users by their usernames.
```

Comment:
nit: Can you fix the grammar here, and add a minimum server version documenting same? (We literally started that yesterday, so not to worry for missing it!)
```suggestion
	// GetUsersByUsernames gets users by their usernames.
	//
	// Minimum server version: 5.6
```